doc/source/configuration/release-train.rst

```diff
@@ -222,7 +222,7 @@ has not yet been deployed. This can be avoided with the following workflow:
 
 .. code-block:: console
 
-   kayobe seed service deploy --tags seed-deploy-containers --kolla-tags none
+   kayobe seed service deploy --tags seed-deploy-containers --kolla-tags none -e deploy_containers_registry_attempt_login=false
```

Comment:
We set `seed_deploy_containers_registry_attempt_login=false` in seed.yml. Perhaps you were missing it in your config?

Reply:
It's set in Zed but not Antelope. Should get merged up today: https://github.com/stackhpc/stackhpc-kayobe-config/pull/997

Reply:
thanks @markgoddard - for clarity:
> We set seed_deploy_containers_registry_attempt_login=false in seed.yml

do you mean 'we set this after the fact' or 'this is set by default in seed.yml.   I don't have it in seed.yml, but also can't find it set in the seed.yml in this repo.

Reply:
ah, thanks for the clarification @Alex-Welsh .   @markgoddard ignore my question then!

Reply:
I've pulled in the latest changes, and I no longer need the extra var, so this change isn't needed.   I'll close off.

Thanks all!